cypress/support/pages/admin_console/Masthead.ts

```diff
@@ -64,7 +64,9 @@ export default class Masthead extends CommonElements {
   }
 
   closeAllAlertMessages() {
-    cy.get(this.closeAlertMessageBtn).click({ force: true, multiple: true });
+    cy.get(this.closeAlertMessageBtn).each(() => {
+      cy.get(this.closeAlertMessageBtn).click({ force: true, multiple: true });
+    });
```

Comment:
I believe the following code (same as before) should yield the same result. At least according to the [Cypress documentation](https://docs.cypress.io/api/commands/click#Click-all-elements-with-id-starting-with-btn).
```suggestion
    cy.get(this.closeAlertMessageBtn).click({ force: true, multiple: true });
```

Reply:
I've tried this way. But the problem is. When Cypress perform cy.get(this.closeAlertMessageBtn).click({ force: true, multiple: true });  it finds multiple elements present, after first notification is closed. Dom is updated, and it loses other elements. So in this case it will only close last notification, but not all of them. So after each click, I have to find again the elements.

Reply:
Ahhh, yeah that makes sense. Let's keep it like this then.